src/MongoDbOperationStore.ts

```diff
@@ -0,0 +1,139 @@
+import { Config, ConfigKey } from './Config';
+import { Collection, Db, MongoClient } from 'mongodb';
+import { Operation } from './Operation';
+import { OperationStore } from './OperationStore';
+
+/**
+ * Sidetree operation stored in MongoDb.
+ */
+interface MongoOperation {
+  didUniqueSuffix: string;
+  operationBufferBase64: string;
+  operationIndex: number;
+  transactionNumber: number;
+  transactionTime: number;
+  batchFileHash: string;
+}
+
+/**
+ * Implementation of OperationStore that stores the operation data in
+ * a MongoDB database.
+ */
+export class MongoDbOperationStore implements OperationStore {
+  private serverUrl: string;
+  private client: MongoClient | undefined;
+  private db: Db | undefined;
```

Comment:
`db` and `client` are still both class members. Consider moving them into `initialize()`.

Reply:
Moved them to initialize()